deploy/common/k8sbased.sh

```diff
@@ -583,17 +591,28 @@ function launch_sensor {
         )
       fi
 
+      local helm_chart="$k8s_dir/sensor-deploy/chart"
+
       if [[ -n "$CI" ]]; then
-        helm lint "$k8s_dir/sensor-deploy/chart"
-        helm lint "$k8s_dir/sensor-deploy/chart" -n stackrox
-        helm lint "$k8s_dir/sensor-deploy/chart" -n stackrox "${helm_args[@]}" "${extra_helm_config[@]}"
+        helm lint "${helm_chart}"
+        helm lint "${helm_chart}" -n stackrox
+        helm lint "${helm_chart}" -n stackrox "${helm_args[@]}" "${extra_helm_config[@]}"
       fi
+
       if [[ "$sensor_namespace" != "stackrox" ]]; then
         kubectl create namespace "$sensor_namespace" &>/dev/null || true
         kubectl -n "$sensor_namespace" get secret stackrox &>/dev/null || kubectl -n "$sensor_namespace" create -f - < <("${common_dir}/pull-secret.sh" stackrox docker.io)
       fi
-      helm upgrade --install -n "$sensor_namespace" --create-namespace stackrox-secured-cluster-services "$k8s_dir/sensor-deploy/chart" \
+
+      if [[ -n "${SENSOR_CHART_DIR_OVERRIDE}" ]]; then
+        helm_chart="${SENSOR_CHART_DIR_OVERRIDE}"
+      fi
```

Comment:
Similar suggestions as for Central chart, see (**).

Reply:
Okay